routers/api/v1/convert/convert.go

```diff
@@ -224,12 +224,15 @@ func ToUser(user *models.User, signed, admin bool) *api.User {
 	result := &api.User{
 		ID:        user.ID,
 		UserName:  user.Name,
-		AvatarURL: user.AvatarLink(),
 		FullName:  markup.Sanitize(user.FullName),
+		AvatarURL: user.AvatarLink(),
+		Language:  user.Language,
 		IsAdmin:   user.IsAdmin,
 	}
 	if signed && (!user.KeepEmailPrivate || admin) {
 		result.Email = user.Email
+	} else {
+		result.Email = ""
```

Comment:
result.Email will be "" by default. So this is unnecessary. Go defaults unset strings to "".

Reply:
> result.Email will be "" by default. So this is unnecessary. Go defaults unset strings to "".

can you tell me why return fake mail by default, it's very confused for end user. should I return `u.getEmail()` too?